apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/StartSpanOptions.cs

```diff
@@ -22,16 +25,22 @@ public sealed class StartSpanOptions
         /// <summary>Gets the span kind.</summary>
         public SpanKind SpanKind { get; }
 
-        private StartSpanOptions(SpanKind spanKind)
+        /// <summary>Gets the lables to be added to the span.</summary>
```

Comment:
Done.

I updated the PR comment. We need this change for the new RunIn functions as otherwise its not as easy to pass in labels to the spans and would require the user to do it in the function/action being traced.